config/locales/en/topical_events.yml

```diff
@@ -10,3 +10,6 @@ en:
       latest: Latest
       publications: Publications
       travel_advice: Travel Advice
+      travel_advice: "Travel Advice"
+    publications: "Publications"
```

Comment:
It looks like the `publications` key has crept into this PR, probably because of rebasing.